### DISCUSSION TITLE
Welcome to mpc-qt Discussions!

### DISCUSSION THREAD

**Discussion by cmdrkotori** [mpc-qt/mpc-qt#155] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect and collaborate with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Ammar-Sadaoui** [comment]
I'm really impressed with Mpc-Qt! It's amazing how you've managed to recreate the interface and functionality of MPC-HC in a cross-platform application using Qt, while also utilizing libmpv for media playback instead of DirectShow. I'm curious to know, what inspired you to embark on this project? Was there a particular aspect of MPC-HC or Qt that motivated you to bring this vision to life? Thanks for your hard work and contribution to the open-source community!"

**Comment by cmdrkotori** [comment]
>I'm curious to know, what inspired you to embark on this project?

At the time there were Windows trolls online who said that Linux did not have a media player because it didn't have MPC-HC. This was very annoying, so I set about filling that gap.

>Was there a particular aspect of MPC-HC or Qt that motivated you to bring this vision to life?

Up until this point I had made tiny little Qt projects to scratch certain itches on my desktop, so eventually making a larger Qt project was a natural progression. My first foray into making an mpv front-end was [mplaylist](https://github.com/cmdrkotori/mplaylist), because I was having problems that made selecting the next episode in the file manager difficult. When I started this project my goal was to make a media player that looked and acted like MPC-HC, had automatic playlist progression to help with binge watching, and had the playlist robustness of clementine because that was the music player I was using at the time.

**Comment by amayra** [comment]
If I have multiple issues or small feature requests, should I open one issue and include everything in it, or create separate issues?

**Reply by cmdrkotori** [reply]
Separate issues.

**Comment by gladtobeblazed** [comment]
MPC is my favorite media player of all time! The only thing stopping me from switching from SMPlayer is lack of SmoothVideo Project support. It looks like it worked at some point in the past (before I switched from Windows to Linux) but not anymore. Was this an issue with SVP or with MPC-QT?

**Reply by cmdrkotori** [reply]
It still works.
![Screenshot_20241013_181135](https://github.com/user-attachments/assets/61b14832-8f82-44b6-b696-cc279b786134)

You have to change two settings in SVPManager to point to mpc-qt's binary and mpv interface.
![Screenshot_20241013_181406](https://github.com/user-attachments/assets/77df84f7-c99e-45e9-853b-6bc7bd3b2860)

I'm using Arch and have svp installed with svp-bin(AUR), and mpc-qt installed via mpc-qt-git(AUR).

**Comment by gladtobeblazed** [comment]
I see what I did wrong, I kept it as /tmp/mpvsocket since that works with SMPlayer and VLC.  Changing it to /tmp/cmdrkotori.mpc-qt.mpv makes it work, and it somehow still works with those other players! Why it still works is beyond me, but thanks a lot!

edit in case people are confused: use /tmp/cmdrkotori.mpc-qt.mpv and /usr/bin/mpc-qt don't just change one or the other, change both.

**Comment by vahanar** [comment]
Hi,
Sorry i m not a native english speaker
I used mpc on windows to sort video records, and a simple option is missing, is delete

**Reply by tbertels** [reply]
Are you talking about #374?

**Reply by vahanar** [reply]
yes, it's that, deleting file with press del key
i take a lot of video with my smartphone, ( 2To sd card ) and sometime i sort it, but it's a mess
open video > forward > forward > next or del  > forward > forward > next or del > etc ...

**Comment by tbertels** [comment]
@cmdrkotori You may want to go to https://github.com/orgs/mpc-qt/people and mark your membership as public.
This way, you'll be shown as a maintainer/member of mpc-qt for everyone (not just you and me, and not just in Discussions) in the tooltip shown when hovering your name while in the project, on your profile page and on https://github.com/mpc-qt.

**Reply by cmdrkotori** [reply]
Thanks for the heads up.
